components/landing/TheQuickStart/Algorithms.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <section class="algorithms">
+    <div class="algorithms__container">
+      <h3>Get the SDK</h3>
+      <div class="algorithms__section">
+        <CodeSnippet
+          :code-lines="installCode"
+          :code-snippet-title="'TODO: update this'"
+          :code-snippet-location="'quick-start-algorithms'"
+        />
+      </div>
+      <h3>Test some algorithms</h3>
+      <bx-tabs container :value="activeTabValue" @bx-tabs-selected="updateSelectedTab($event)">
+        <bx-tab
+          v-for="(algorithm, index) in codeExamples"
+          :key="index"
+          :target="index"
+          :value="index"
```

Comment:
Do we need to pass the `index` of the elements? Maybe we could just use `algorithm.name` instead since it must be unique.

This way we could avoid adding an extra variable.